src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -136,9 +135,9 @@ public void scrollToElementInModal(WebElement element, WebElement modal) {
   }
 
   public void scrollElementIntoViewPort(WebElement element) {
-    if (!isElementInViewPort(element)) {
+    //if (!isElementInViewPort(element)) {
       scrollToElement(element);
-    }
+    //}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)